plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2597,6 +2597,7 @@ public void testLocalDynamicFilteringWithSelectiveBuildSizeJoin()
 
         session = Session.builder(getSession())
                 .setSystemProperty(JOIN_DISTRIBUTION_TYPE, BROADCAST.name())
+                .setCatalogSessionProperty(ICEBERG_CATALOG, "dynamic_filtering_wait_timeout", "3s")
```

Comment:
Can we make this 1hr and put a timeout of 30 sec on the test method ?

Reply:
Sure